app/views/shared/poly_metadata/edit/_study.html.erb

```diff
@@ -0,0 +1,30 @@
+<%# This partial contains poly_metadata form fields for a study. It is included 
```

Comment:
So is this polymetadata field entry showing for ALL study edit forms? For an scRNA-only pipeline-specific field? Not sure that's a good idea, studies aren't supposed to know anything about specific pipelines.

Reply:
I can't find a way of finding out which study is aimed for scRNA Core. That is why I created the "Additional Properties" section to separate it from other options.

An alternative could be creating a "Pipeline Options" link on the sidebar and show all Pipeline related options for the study in edit/show in a separate page. Actually, I tried this initially and turned out a bit more complicated (creating new controller, views, routes, etc.) Should we do this instead?

Reply:
One of the reasons I abandoned the idea of "Pipeline Options" is that, it sounds rather limiting instead of just any generic options that can be listed under "Additional Properties".

Reply:
I have added the small text "Following fields are optional and specific to certain studies" to where additional properties appear.